sdcm/stress/base.py

```diff
@@ -174,6 +178,131 @@ def _build_log_file_id(loader_idx, cpu_idx, keyspace_idx):
         keyspace_suffix = f"-k{keyspace_idx}" if keyspace_idx else ""
         return f"l{loader_idx}-c{cpu_idx}{keyspace_suffix}-{uuid.uuid4()}"
 
+    def get_stress_usage_text(self, loader, option):
+        result = loader.run(
+            cmd=f'{self.stress_tool_name} help {option} | grep "^Usage:"',
+            timeout=self.timeout,
+            ignore_status=True).stdout
+        LOGGER.debug("Available suboptions for '%s', '%s': %s", self.stress_tool_name, option, result)
+        return result
+
+    @staticmethod
+    def _parse_help_text(result):
```

Comment:
Such functions must be placed in each benchmarking tool related module/class, not in common place.
Common place must have some placeholder if not critical to lack the proper implementatiton.